nixos/modules/services/web-apps/plausible.nix

```diff
@@ -195,6 +244,35 @@ in {
             # Configuration options from
             # https://plausible.io/docs/self-hosting-configuration
             PORT = toString cfg.server.port;
+            LISTEN_IP = cfg.server.listenAddress;
+            # Elixir spwans the Erlang VM, which will listen by default on all
+            # interfaces for messages between Erlang nodes (capable of)
+            # remote code execution; it can be protected by a cookie; see
+            # https://erlang.org/doc/reference_manual/distributed.html#security).
+            # To restrict this to an interface of our choice (e.g. localhost
+            # or a VPN IP) we need to set `inet_dist_use_interface`,
+            # which accepts the IP address as an Erlang tuple.
+            RELEASE_VM_ARGS =
+              let
+                ip = cfg.erlang.vmListenAddress;
+                # Elixir script that turns an IP address into an Erlang address tuple.
+                # Also works on IPv6.
+                # Example: `127.0.0.1` -> `{127,0,0,1}`.
+                erlangAddressTupleCommand = pkgs.writeTextFile {
+                  executable = true;
+                  name = "ip-to-erlang-address-tuple.exs";
+                  text = ''
+                    #! ${pkgs.elixir}/bin/elixir
+                    {:ok, addr} = :inet.parse_address('${ip}')
+                    IO.puts(inspect(addr))
```

Comment:
Remaining debug statement?

Reply:
No, that's the thing that prints the Erlang-formatted IP address to stdout.

Reply:
I've made that clearer with a comment.